lib/util/cache/package/redis.ts

```diff
@@ -102,6 +102,7 @@ export async function init(
         return Math.min(retries * 100, 3000);
       },
     },
+    pingInterval: 10000,
```

Comment:
```suggestion
    pingInterval: 10000, // 10s
```

@rarkins should we raise it to 30s?